plugins/do-build.py

```diff
@@ -42,6 +42,12 @@ def make_simx(args):
 
     subprocess.call(cmd + ['-j%d' %(args.num_jobs)])
 
+def make_rdmo_app(args):
```

Comment:
I started throwing in some stuff to make MKT build my rdmo app. I abandoned that, though. Ignore references to rdmo-app and the packages added to support.Dockerfile.

Reply:
Building inside VM looks simpler, but it misses the MKT concept. We wanted to separate build environment from run environment. It allows us to enjoy from specific optimizations and makes run fast.